### ISSUE TITLE
RunDEtest function for differential expression analysis between two genotypes from the same cell type

### ISSUE THREAD

**Issue by shuaifu93** [zhanghao-njmu/SCP#57]
Thanks for creating this cool tool for scRNA-seq analysis, I have now integrated control and patient sample using harmony,  is there any way we can do to perform differential expression analysis between disease vs control from the same cell type using the RunDEtest function?  When I use findmarkers from Seurat package, I would use 
DEG <- FindMarkers(integrated_dataset,   assay = "RNA",  min.pct = 0.1, ident.1 = "disease", ident.2 = "control",
                               test.use = "wilcox").

For RunDEtest function of your package, is the code below works the same purpose?
integrated_dataset <- RunDEtest(srt = integrated_dataset, group_by = "orig.ident", group1 = "disease", group2 = "control", only.pos = F, fc.threshold = 1, FindAllMarkers = T,  FindPairedMarkers = F)

Thanks.

**Comment by zhanghao-njmu** [comment]
Yes, RunDEtest just modifies the parameters according to the general habits, and still uses Seurat's FindMarkers function internally. Note that some default parameters have been modified, such as norm.method = "LogNormalize", fc.threshold = 1.5, etc.
